language: "ca"
task description: "Buscaré {object_class}."
next location: "Vaig a la següent ubicació."
next location detailed: "El proper lloc on buscaré és {location}."
next location detailed ongoing: "Encara no he trobat {object_class}. Continuaré buscant a {location}."
found object: "He trobat {object_class}."
found object grasp: "He trobat {object_class}. Intentaré recollir {object_class}."
grasp successful: "He recollit {object_class}."
and: "i"
trajectory summary start: "He buscat als següents llocs:"
location not accesible clause: "(on no he pogut accedir)"
location object found clause: "(on he trobat {object_class})"
grasp failed summary: "Vaig intentar recollir {object_class} però vaig fallar."
returning to user: "Tornaré a l'usuari."
deliver object: "Aquí tens {object_class}."
task outcome failure: "No he pogut portar {object_class}."
object not found: "No he trobat {object_class}."
object not found unreachable locations: "No he trobat {object_class}, però no he estat capaç darribar a alguns llocs on podria cercar {object_class}"
object cannot grasp: "No he pogut agafar {object_class}. Ho sento."
object unknown: "No sé com portar-te {object_class}. Prova un altre objecte."
retry grasp: "No vaig poder agafar {object_class}. Ho intentaré de nou."
initial_plan_medium: "Buscaré {object_class}. Començaré a buscar a {location}."
initial_expectations_failure: "Buscaré {object_class}. Normalment ho puc fer, però de vegades podria no detectar {object_class} o el meu camí podria estar bloquejat."
object_class:
  cup: "la tassa"
  bottle: "l'ampolla"
  remote: "el comandament"
  apple: "la poma"
  banana: "el plàtan"
  wine glass: "la copa"
  oven: "el forn"
  orange: "la taronja"
location:
  kitchen: "la cocina"
  bedroom: "el dormitorio"
  table: "la mesa"
  lounge: "la sala de estar"